tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/LGGeneratorTests.cs

```diff
@@ -451,6 +457,14 @@ public async Task TestNoResourceExplorerLanguageGeneration()
             .StartTestAsync();
         }
 
+        internal static void Initialize()
+        {
+            ComponentRegistration.Add(new DeclarativeComponentRegistration());
+            ComponentRegistration.Add(new AdaptiveComponentRegistration());
+            ComponentRegistration.Add(new AdaptiveTestingComponentRegistration());
+            ComponentRegistration.Add(new LanguageGenerationComponentRegistration());
+        }
```

Comment:
This only should be run once, not with each unit test.  Old test was [AssemblyInitialize]

Reply:
updated